### PR TITLE
Recipe for pykalman

### PR DESCRIPTION
**Pull request by pmlandwehr**
The recipe only specifies a "BSD" license, as matches the docs on pypi. (Seems like a good question for @duckworthd) since he's the lead developer.

### CODE REVIEW

File: recipes/pykalman/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  imports:
+    - pykalman
+    - pykalman.datasets
+    - pykalman.sqrt
+    - pykalman.sqrt.tests
+    - pykalman.tests
+
+about:
+  home: http://pykalman.github.io
+  license: BSD License
```
Comment (ocefpaf) [inline recipes/pykalman/meta.yaml]: I believe it is `BSD 3-Clause`. (See https://github.com/pykalman/pykalman/blob/master/COPYING.)

**Comment by jakirkham** [comment]
Nice addition!

File: recipes/pykalman/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/pykalman/meta.yaml]: Added a space @ocefpaf as the `.` was getting added to link. 😄

> I believe it is `BSD 3-Clause`.

👍

File: recipes/pykalman/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  url: https://pypi.io/packages/source/p/pykalman/pykalman-{{ version }}.tar.gz
+  md5: 8baa2921cdfd2a5193ac67e23c2f7e5e
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
```
Comment (ocefpaf) [inline recipes/pykalman/meta.yaml]: Can you add `setuptools` explicit here. Otherwise `--single-version-externally-managed` might fail in the future.

**Comment by pmlandwehr** [comment]
Hmm. Not sure why this can't be merged. Anyone?

**Comment by jakirkham** [comment]
Not sure either. Could you please try updating your copy of `master` and rebase or merge this branch. Admittedly this may not be possible, but I'm remote so can't easily check. If that's the case, try toggling the open status of the branch. Sometimes GitHub gets confused and that should set help set it straight.

**Comment by jakirkham** [comment]
Going to try my own suggestion. 😜

**Comment by jakirkham** [comment]
Nope. Please try rebasing or merging, @pmlandwehr.

File: recipes/pykalman/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    - numpy
+    - scipy
+
+test:
+  imports:
+    - pykalman
+    - pykalman.datasets
+    - pykalman.sqrt
+    - pykalman.sqrt.tests
+    - pykalman.tests
```
Comment (patricksnape) [inline recipes/pykalman/meta.yaml]: This imports the tests but I assume doesn't run them? Should we run some tests?

Comment (ocefpaf) [inline recipes/pykalman/meta.yaml]: That happens in a lot of our feedstocks due to the `conda-skeleton` boilerplate. Let's get this into a feedstock and the discussion about further testing can happen there.

**Comment by jakirkham** [comment]
Restarted CI as we were rate limited previously. This is now out in the wild.

**Comment by jakirkham** [comment]
@duckworthd, just letting you know that we have added `pykalman` to [conda-forge]( http://conda-forge.github.io/ ). This will allow users to easily install this with [`conda`]( http://conda.pydata.org/docs/ ) ( [`conda install -c conda-forge pykalman`]( http://anaconda.org/conda-forge/pykalman ) ). The maintenance burden here is light as all releases are built and deployed automatically with CI. Changes to how it is being built can be controlled by changing the recipe in [this feedstock (repo)]( https://github.com/conda-forge/pykalman-feedstock ). If you have any interest in being added as a maintainer (useful for making new releases for instance), we would be more than happy to add any of you to the maintainers list.

**Pull request merged**